settings/scripts/Create multisite workflow.r

```diff
@@ -0,0 +1,26 @@
+rm(list=ls())
```

Comment:
? This just clears the workspace. This file also goes in the category of "should probably remove from the package." I wanted you to see an example of how I'm using this, but didn't really intend for this to be widely used. I guess it should be a man page example or in a vignette then?